src/components/common/Directors.vue

```diff
@@ -588,51 +581,60 @@ export default class Directors extends Vue {
   @Getter(useBusinessStore) getIdentifier!: string
   @Getter(useBusinessStore) getLastAnnualReportDate!: string
   @Getter(useBusinessStore) getLastDirectorChangeDate!: string
-  @Getter(useBusinessStore) isBenBcCccUlc!: boolean
 
   /** Effective date for fetching and appointing/ceasing directors. */
-  private asOfDate: string
+  asOfDate: string
 
   /** Original directors from Legal API. */
-  private original: DirectorIF[] = []
+  original: DirectorIF[] = []
 
   /** Working directors data (also used as v-model for the HTML list). */
-  private allDirectors: DirectorIF[] = []
-
-  private directorEditInProgress = false
-  private directorPreEdit = null // officer before edit
-  private showNewDirectorForm = false
-  private showPopup = false
-  private activeIndex = -1
-  private activeIndexCustomCease = -1
-  private activeDirectorToDelete: DirectorIF = null
-  private cessationDateTemp: string = null
-  private isEditingDirector = false
-  private messageIndex = -1
+  allDirectors: DirectorIF[] = []
+
+  /** Initialize dir to use in the template's for loop. */
+  dir: DirectorIF = {
```

Comment:
This is a decoy fix, which calms the tests perhaps, but doesn't solve the problem.

I see on line 191 that `dir` is a loop variable. Maybe try renaming it in case it's a reserved word? You could also try moving the v-for to a template element around the `<li>`.

Reply:
Both didn't work Sev. I was thinking of adding optional chaining to dir in the template. Apparently, it's not supported in Vue 2. It is in Vue 3: https://github.com/vuejs/vue/issues/11088

I'm thinking of disabling the tests for the time being and adding them to the new ticket. I'll look into this more and try to fix before doing so.

Reply:
Does the code work correctly right now?

If it does then create a follow-up ticket with the comments above and move on.

Reply:
I've been messing around with the Directors and I see everything is working correctly with no errors. So, I guess the code is OK.

Alright, got it.